web/frontend/components/CollapseTriangle.vue

```diff
@@ -1,6 +1,6 @@
 <template>
-  <svg height="32" width="32" v-bind:class="{open: !collapsed, collapsed:collapsed}">
-    <polygon points="6,6 20,16 6,24" />
+  <svg height="20" width="32" v-bind:class="{open: !collapsed, collapsed:collapsed}">
```

Comment:
Curious about this change — was the height of the svg specifically an issue?

Reply:
So, I was trying to get everything aligned, and kind of went on a journey...

![image](https://github.com/harvard-lil/h2o/assets/11020492/8a40fc8c-a9a2-4226-bf8e-3dcd89cb2d99)

After the delete button is made to line up, this SVG's box was sufficiently large to affect the overall height of the TOC entry.... such that the black "section" entries, which have the expand/collapse triangle were taller than the white "resource" entries.

And, I noticed that the triangle itself wasn't filling up the box: the box had a lot of empty space in it.... which also made that hard to visually align (evidently leading to those -8px margins etc. that I was able to remove.)

I did make this change BEFORE figuring out `align-self:center;` and removing some of the other CSS..... so it might be that I didn't need to do this.